### PR TITLE
[plugin.video.rsiplaytv] v1.5.0

### PR DESCRIPTION
**Pull request by goggle**
### Description
Release of `plugin.video.rsiplaytv` v1.5.0 in the Leia repository (because Krypton does not allow any new plugins, see the previous PR in the Krypton branch: https://github.com/xbmc/repo-plugins/pull/2338). 

The requested code review changes are included by this commit (https://github.com/goggle/plugin.video.rsiplaytv/commit/8dc613805136d3c495243d415e23fd4dbf1d4524).

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: plugin.video.rsiplaytv/addon.xml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.rsiplaytv" name="RSI Play TV" version="1.1.0" provider-name="Alexander Seiler">
+  <requires>
+    <import addon="xbmc.python" version="2.25.0"/>
```
Comment (Rechi) [inline plugin.video.rsiplaytv/addon.xml]: > WARN: For leia it is advised to set xbmc.python version to 2.26.0
```suggestion
    <import addon="xbmc.python" version="2.26.0"/>
```

File: plugin.video.rsiplaytv/resources/lib/rsiplaytv.py [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+    from urllib import unquote_plus
+
+from kodi_six import xbmc, xbmcaddon, xbmcplugin
+import srgssr
+
+ADDON_ID = 'plugin.video.rsiplaytv'
+REAL_SETTINGS = xbmcaddon.Addon(id=ADDON_ID)
+ADDON_NAME = REAL_SETTINGS.getAddonInfo('name')
+ADDON_VERSION = REAL_SETTINGS.getAddonInfo('version')
+DEBUG = (REAL_SETTINGS.getSetting('Enable_Debugging') == 'true')
```
Comment (Rechi) [inline plugin.video.rsiplaytv/resources/lib/rsiplaytv.py]:
```suggestion
DEBUG = REAL_SETTINGS.getSettingBool('Enable_Debugging')
```

File: plugin.video.rsiplaytv/resources/lib/rsiplaytv.py [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+    try:
+        page = unquote_plus(params['page'])
+    except Exception:
+        page = None
+
+    log('Mode: ' + str(mode))
+    log('URL : ' + str(url))
+    log('Name: ' + str(name))
+    log('Page Hash: ' + str(page_hash))
+    log('Page: ' + str(page))
```
Comment (Rechi) [inline plugin.video.rsiplaytv/resources/lib/rsiplaytv.py]: combine this into one log line

**Comment by goggle** [comment]
@Rechi I'm sorry, I must have done something wrong in my git workflow... Will fix this immediately.

**Pull request merged**